frontend/crd/src/app/security/user.ts

```diff
@@ -0,0 +1,5 @@
+export interface TempUser {
```

Comment:
did you add all the temp user stuff or is this from a different PR that hasn't been merged yet?

Reply:
Yeah this stuff was mine. I'm pretty sure these are just merge artifacts